src/lib/viewers/BaseViewer.js

```diff
@@ -1122,6 +1122,11 @@ class BaseViewer extends EventEmitter {
                 this.containerEl.classList.add(className);
             }
         }
+
+        if (mode === AnnotationMode.DRAWING) {
```

Comment:
Is this condition needed or can emit it always? Do we emit this event anywhere else that can be centralized here?

Reply:
We can emit it always.

We emit this event in `handleAnnotationColorChange` in DocBaseViewer and ImageViewer but I think those should be separate from this as `handleAnnotationColorChange` is tied to the user action of clicking on a color in the ColorPalette. The event that is emitted in BaseViewer is more so tied to the user action of when a user is clicking on the Drawing button to show the Color Picker.